app/models/gobierto_plans/plan.rb

```diff
@@ -35,6 +35,14 @@ def nodes
       @nodes ||= Node.joins(categories: [:vocabulary]).where(terms: { vocabulary_id: categories_vocabulary&.id }).order(position: :asc, id: :asc)
     end
 
+    def have_indicators?
+      GobiertoCommon::CustomFieldRecord.includes(:custom_field).where(custom_fields: { instance_id: id, field_type: 10 }).count > 0
+    end
+
+    def indicators
+      GobiertoCommon::CustomFieldRecord.includes(:custom_field).where(custom_fields: { instance_id: id, field_type: 10 })
+    end
```

Comment:
It's clearer to use the symbol or string associated to an enum attribute instead of its numeric value, use `field_type: :plugin`

The indicators belongs to custom fields of `type` `plugin` which also have `plugin_type` `table`. You should include the plugin type in the query to avoid errors, with something like:
```
GobiertoCommon::CustomFieldRecord.includes(:custom_field).where(
  custom_fields: { instance_id: id, field_type: :plugin }
).where(
  "#{GobiertoCommon::CustomField.table_name}.options @> ?", { configuration: { plugin_type: :table } }.to_json
)
```